assets/javascripts/search.js

```diff
@@ -0,0 +1,218 @@
+$(document).ready(function() {
```

Comment:
There's a shorthand for this. Any function given directly to the jQuery function is called on document ready.

[Reference](https://api.jquery.com/jQuery/#jQuery3)